### ISSUE TITLE
Create the tickets

### ISSUE THREAD

**Issue by Xymmer** [SAP/cloud-commerce-spartacus-storefront#585]
Miguel submit list to Bill
Bill to create tickets
We'll start doing a standup every few days

**Comment by developpeurweb** [comment]
Hey @Xymmer below you'll find the description, same one for each ticket, and the list.
I only broke-down pages/components bigger than 4 files (usually a component has a paired html and scss file).
As Tobias mentioned Deno is already working on the Header and Louis in 2 components of the PDP, I've flagged those for you with an _(In Progress by ...)_ comment.
I'm probably missing other files, especially from the `cxbase` folder, but with these below I think we have enough to start and we'll add more as we find them.

**Description for each ticket:**

_In SCSS files_: 
Remove BEM selectors and create new simple ones
Add styles to selectors using the official CSS variables list found in:
`projects/storefrontlib/src/styling/_variables.scss`
Remove SASS variables only if needed, keeping in mind clients cannot change them

_In HTML files:_
Remove BEM classes with simple ones
keep it simple, keep semantic name of the element, remove `<div>`s that are not needed

**List of tickets**

Footer refactor (4 files)
Add to cart refactor (4 files)
Product Carousel refactor (2 files)
Order Confirmation refactor (2 files)
My Account - Order History list refactor (2 files)
My Account - Order Details refactor (2 files)
My Account - Payment Methods refactor (2 files)

_(In Progress by Deno)_
Header refactor / language-selector component
Header refactor / currency-selector component
Header refactor / tertiary-bar component
Header refactor / login component
Header refactor / search-box component
Header refactor / mini-cart component
Header refactor / category-navigation component
Header refactor / mobile-menu component

Login refactor / login-form component
Login refactor / register component
Login refactor / reset password component
Login refactor / reset new password component

Search Results - Category Listing / product-list component
Search Results - Category Listing  / product-facet-navigation component
Search Results - Category Listing  / product-grid-item component
Search Results - Category Listing  / product-list-item component
Search Results - Category Listing  / product-view component
Search Results - Category Listing / category-page-layout component

Product Details Page / product-details component
Product Details Page / product-attributes component
Product Details Page / product-images component _(In Progress by Louis)_
Product Details Page / product-reviews component _(In Progress by Louis)_
Product Details Page / product-summary component 

Cart Page refactor / add-to-cart component
Cart Page refactor / added-to-cart component
Cart Page refactor / cart-details component
Cart Page refactor / cart-item component
Cart Page refactor / cart-item-list component
Cart Page refactor / order-summary component

Checkout Wizard refactor / multi-step-checkout component
Checkout Wizard refactor / delivery-mode component
Checkout Wizard refactor / payment-method component
Checkout Wizard refactor / payment-form component
Checkout Wizard refactor / review-submit component
Checkout Wizard refactor / shipping-address component
Checkout Wizard refactor / address-form component
Checkout Wizard refactor / suggested-addresses-dialog component

Store Finder refactor / schedule component
Store Finder refactor / store-finder-grid component
Store Finder refactor / store-finder-header component
Store Finder refactor / store-finder-list-item component
Store Finder refactor / store-finder-map component
Store Finder refactor / store-finder-search component
Store Finder refactor / store-finder-list component
Store Finder refactor / store-finder-search-result component
Store Finder refactor / store-finder-store-description component
Store Finder refactor / store-finder-stores-count component
